CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 - Support for tracing Faraday requests ([#2345](https://github.com/getsentry/sentry-ruby/pull/2345))
   - Closes [#1795](https://github.com/getsentry/sentry-ruby/issues/1795)
   - Please note that the Faraday instrumentation has some limitations in case of async requests: https://github.com/lostisland/faraday/issues/1381
+- Support for attachments ([#2357](https://github.com/getsentry/sentry-ruby/pull/2357))
```

Comment:
for new features, we add a small snippet below the changelog entry that shows users how to use the new feature. So here,

```ruby
Sentry.add_attachment(path: '/foo/bar.txt')
Sentry.add_attachment(filename: 'payload.json', bytes: '{"value": 42}'))
```